src/lsm/groove.zig

```diff
@@ -117,7 +117,7 @@ comptime {
 fn IndexTreeType(
     comptime Storage: type,
     comptime Field: type,
-    comptime value_count_max: usize,
+    comptime table_value_count_max: usize,
```

Comment:
This was a confusing name! This is not `value_count_max` for the entire tree, just for the table!